ckanext/scheming/validation.py

```diff
@@ -569,14 +569,29 @@ def validator(key, data, errors, context):
     return validator
 
 
+def comma_swap_formatter(arg_value):
+    """
+    Swaps the parts of a string around a single comma.
+    Use to format e.g. "Tanzania, Republic of" as "Republic of Tanzania"
+    """
+    if arg_value.count(',') == 1:
```

Comment:
How about `if ',' in arg_value:` . I usually see it this way in python.

Reply:
I want to make sure only 1 comma - if there are two commas, this logic doesn't make sense.